### DISCUSSION TITLE
Charging condition before drive cycle?

### DISCUSSION THREAD

**Discussion by robbiemacinnes** [pybamm-team/liionpack#275] [Q&A] [answered]
Is it possible to simulate an experiment which begins with static charging and then is discharged using a drive cycle in liionpack?

I know it's possible in PyBaMM using the .step.current etc or by initialising the model using a set of initial conditions however, when I attempt this in a number of ways using Liionpack, I recieve this error each time.

AttributeError: 'Experiment' object has no attribute 'operating_conditions'

Could be a simple fix that I am missing, any help would be greatly appreciated. Thanks

**Comment by TomTranter** [comment, marked as answer]
Hey, sorry for the late reply. What you're trying to do should be possible but the codebase is a little broken right now. You could roll back the PyBaMM version to the previous major release and try again. Linked to issue #276

**Reply by robbiemacinnes** [reply]
Thanks for the reply. Yeah, that is what I ended up doing. Reverting back to an older version of PyBaMM works fine

**Reply by lappemic** [reply]
@robbiemacinnes what version did you end up using? I get other version conflicts if i change the pybamm version.

**Reply by lappemic** [reply]
Got it resolved - with PyBaMM v23.4 it is currently working without issues. Hope this helps anyone else encountering the same problem!
